<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>adminSetUpWindow</class>
 <widget class="QDialog" name="adminSetUpWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>675</width>
    <height>650</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>135</x>
     <y>80</y>
     <width>131</width>
     <height>40</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>16</pointsize>
    </font>
   </property>
   <property name="text">
    <string>company name:</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="companyNameInput">
   <property name="geometry">
    <rect>
     <x>290</x>
     <y>80</y>
     <width>250</width>
     <height>40</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>135</x>
     <y>150</y>
     <width>131</width>
     <height>40</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>16</pointsize>
    </font>
   </property>
   <property name="text">
    <string>mobile phone: </string>
   </property>
  </widget>
  <widget class="QLineEdit" name="mobilephoneInput">
   <property name="geometry">
    <rect>
     <x>290</x>
     <y>150</y>
     <width>250</width>
     <height>40</height>
    </rect>
   </property>
  </widget>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>350</y>
     <width>405</width>
     <height>111</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>16</pointsize>
    </font>
   </property>
   <property name="title">
    <string>Supported Payment Methods:</string>
   </property>
   <widget class="QCheckBox" name="visaCheckBox">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>40</y>
      <width>85</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Visa </string>
    </property>
   </widget>
   <widget class="QCheckBox" name="cashCheckBox">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>80</y>
      <width>85</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Cash</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="instaPayCheckBox3">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>40</y>
      <width>85</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>InstaPay</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="fawryCheckBox_2">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>80</y>
      <width>85</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>PayPal</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="onlineWalletCheckBox">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>40</y>
      <width>131</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Oniline Wallet</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="fawryCheckBox">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>80</y>
      <width>85</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Fawry</string>
    </property>
   </widget>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>135</x>
     <y>220</y>
     <width>141</width>
     <height>40</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>16</pointsize>
    </font>
   </property>
   <property name="text">
    <string>company's adrress</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="companyAddressInput">
   <property name="geometry">
    <rect>
     <x>290</x>
     <y>220</y>
     <width>250</width>
     <height>40</height>
    </rect>
   </property>
  </widget>
  <widget class="QGroupBox" name="groupBox_2">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>470</y>
     <width>405</width>
     <height>81</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>16</pointsize>
    </font>
   </property>
   <property name="title">
    <string>Supported Shipping Methods</string>
   </property>
   <widget class="QWidget" name="verticalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>20</y>
      <width>401</width>
      <height>61</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QCheckBox" name="fastShippingCheckBox">
         <property name="text">
          <string>fast shipping </string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QCheckBox" name="ordinaryShippingCheckBox">
         <property name="text">
          <string>ordinary shipping </string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <item>
        <widget class="QLabel" name="label_5">
         <property name="text">
          <string>24h-48h</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_4">
         <property name="text">
          <string>(3-5)days</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QLabel" name="label_6">
   <property name="geometry">
    <rect>
     <x>135</x>
     <y>290</y>
     <width>141</width>
     <height>40</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>16</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Company's email:</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="companyEmailAddressInput">
   <property name="geometry">
    <rect>
     <x>290</x>
     <y>290</y>
     <width>250</width>
     <height>40</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="registerCompanyPushButton">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>561</y>
     <width>405</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>18</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Register your company</string>
   </property>
  </widget>
  <widget class="QLabel" name="companyNameExistsError">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>120</y>
     <width>241</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; color:#bb341f;&quot;&gt;error: company's name already exists&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <widget class="QLabel" name="companyEmailExistsError">
   <property name="geometry">
    <rect>
     <x>290</x>
     <y>330</y>
     <width>341</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; color:#c23621;&quot;&gt;error: company's email is already for another company&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <widget class="QLabel" name="companyPhoneExistsError">
   <property name="geometry">
    <rect>
     <x>290</x>
     <y>190</y>
     <width>341</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; color:#bd3520;&quot;&gt;error: company's phone is already for another company&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <widget class="QLabel" name="allFieldsMustBeCompletedError">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>610</y>
     <width>210</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; color:#cc3922;&quot;&gt;error: all fields must be completed &lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
